hack/common.sh

```diff
@@ -114,6 +114,11 @@ run_WMCO() {
       fi
   fi
 
+  # Add the "openshift.io/cluster-monitoring:"true"" label to the operator namespace to enable monitoring
+  if ! oc patch ns $WMCO_DEPLOY_NAMESPACE -p '{"metadata":{"labels":{"openshift.io/cluster-monitoring":"true"}}}'; then
```

Comment:
I suggest using `oc label ns` rather than patching it given the intent is to only add a label.